tools/ClassGen/InstrGen.cpp

```diff
@@ -300,6 +300,17 @@ int main(int argc, char **argv) {
       .autoVerify(VerifyKind::SameShape, {"Dest", "LHS", "RHS"})
       .autoIRGen("CmpEQ");
 
+  BB.newInstr("ElementIsNaN")
+      .addOperand("Dest", OperandKind::Out)
+      .addOperand("Src", OperandKind::In)
+      .inplaceOperand({
+          "Dest",
+          "Src",
+      })
+      .dataParallel()
+      .autoVerify(VerifyKind::SameType, {"Dest", "Src"})
```

Comment:
dest is bool mask and input is tensor, why sameType? (maybe i'm missing something)